spec/features/merchants/dashboard/index_spec.rb

```diff
@@ -72,4 +76,44 @@
       expect(page).to have_content("Customer Name: Lance B Successful Transactions: 5")
     end
   end
+
+  describe "us-4 Merchant Dashboard Items Ready to Ship" do 
```

Comment:
This test looks great!! However, I think as a team we should discuss as a team if we want to leave in user stories in after we complete them like how you have them in lines #82, #86-89.